### PR TITLE
Refine err msg in SetOffsetAt()

### PR DESCRIPTION
**Pull request by haorenfsa**

### CODE REVIEW

File: reader.go [hunk 1]
```diff
@@ -1090,7 +1093,7 @@ func (r *Reader) SetOffsetAt(ctx context.Context, t time.Time) error {
 
 		return r.SetOffset(offset)
 	}
-	return fmt.Errorf("error setting offset for timestamp %+v", t)
+	return fmt.Errorf("error dailing all brokers, one of the error: %w", err)
```
Comment (achille-roussel) [inline reader.go:1096]:
```suggestion
	return fmt.Errorf("error dialing all brokers, one of the errors: %w", err)
```

File: reader.go [hunk 2]
```diff
@@ -1073,13 +1073,16 @@ func (r *Reader) SetOffsetAt(ctx context.Context, t time.Time) error {
 		return io.ErrClosedPipe
 	}
 	r.mutex.Unlock()
-
+	if len(r.config.Brokers) < 1 {
+		return errors.New("no broker in config")
```
Comment (rhansen2) [inline reader.go:1077]:
```suggestion
		return errors.New("no brokers in config")
```

**Comment by haorenfsa** [comment]
There seems to be some problem with the CI, it may fail on master

**Review by rhansen2 — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**